src/features/reportProblem/ReportProblem.tsx

```diff
@@ -190,33 +216,80 @@ export const ReportProblem = () => {
           <Form.Item
             label="Text"
             name="body"
-            rules={[{ required: true, message: "Please input your problem description" }]}
+            rules={[
+              {
+                required: true,
+                message: "Please input your problem description",
+              },
+            ]}
           >
-            <TextArea rows={5} placeholder="Describe your problem" maxLength={2000} showCount />
+            <TextArea
+              rows={5}
+              placeholder="Describe your problem"
+              maxLength={2000}
+              showCount
+            />
           </Form.Item>
           <Form.Item
             name="upload"
             label="Upload"
             valuePropName="upload"
             getValueFromEvent={normFile}
+            rules={[
+              () => ({
+                validator(_, value) {
+                  if (value.length <= MAX_COUNT_FILES) {
+                    return Promise.resolve();
+                  }
+                  return Promise.reject(
+                    new Error("Only 10 files can be uploaded")
+                  );
+                },
+              }),
+              () => ({
+                validator() {
+                  if (filesSize <= MAX_FILES_SIZE_MB) {
+                    return Promise.resolve();
+                  }
+                  return Promise.reject(
+                    new Error("The maximum file weight cannot exceed 10Mb")
+                  );
+                },
+              }),
+            ]}
             extra={
               <React.Fragment>
-                Extensions: image/*, *.zip, *.rar, *.json, *.txt<br />
-                File size maximum 2Mb, 10 files in total, total size
-                maximum 10Mb
+                Extensions: image/*, *.zip, *.rar, *.json, *.txt
+                <br />
+                File size maximum 2Mb, 10 files in total, total size maximum
+                10Mb
               </React.Fragment>
             }
           >
-            <Upload name="attachments" onChange={handleUploadChange} {...{ fileList }}>
-              <Button
-                icon={
-                  <span role="img" className="anticon anticon-upload">
-                    <UploadSimple />
-                  </span>
+            <Upload
+              name="attachments"
+              onChange={handleUploadChange}
+              {...{ fileList }}
+              multiple={true}
+            >
+              <Tooltip
+                placement="right"
+                title={"Only 10 files can be uploaded"}
+                trigger={
+                  fileList.length >= MAX_COUNT_FILES ? ["hover", "focus"] : ""
                 }
               >
-                Upload
-              </Button>
+                <Button
+                  disabled={fileList.length >= MAX_COUNT_FILES}
```

Comment:
how does this button is disabled when `MAX_FILES_SIZE_MB` exceed?

Reply:
I am not sure that we should disable upload button when MAX_FILES_SIZE_MB exceed. I think this case is missed in requirements.
I can clarify it with Semyon or with someone else